addons/binding/org.openhab.binding.onebusaway/src/main/java/org/openhab/binding/onebusaway/config/ApiConfiguration.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.onebusaway.config;
+
+import static org.openhab.binding.onebusaway.OneBusAwayBindingConstants.*;
+
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+/**
+ * The {@link ApiConfiguration} defines the model for a API bridge configuration.
+ *
+ * @author Shawn Wilsher - Initial contribution
+ */
+public class ApiConfiguration {
```

Comment:
I missed on import one all packages which you do not export (and are thus not meant to be called by others) should be inside the internal package. This applies for example to the `config` and `handler` package.

See [guidelines B.10](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles)